cgi-bin/LJ/Console/Command/EmailAlias.pm

```diff
@@ -68,12 +71,28 @@ sub execute {
         $dbh->do("REPLACE INTO email_aliases VALUES (?, ?)", undef, $alias, $value);
         return $self->error("Database error: " . $dbh->errstr)
             if $dbh->err;
+        if ( $alias ne $alias_with_dashes ) {
+            $dbh->do( "REPLACE INTO email_aliases VALUES (?, ?)",
```

Comment:
Hmmm. I can guarantee you that this will be forgotten by the next time anyone uses this tool but I see your point that this is mostly a manual tool and shouldn't be too clever. 

Can we be a little clever though and present useful information reminding the tool user about underscores etc?




> On Mar 2, 2015, at 10:29, Pau Amma <notifications@github.com> wrote:
> 
> In cgi-bin/LJ/Console/Command/EmailAlias.pm:
> 
> > @@ -68,12 +71,28 @@ sub execute {
> >          $dbh->do("REPLACE INTO email_aliases VALUES (?, ?)", undef, $alias, $value);
> >          return $self->error("Database error: " . $dbh->errstr)
> >              if $dbh->err;
> > +        if ( $alias ne $alias_with_dashes ) {
> > +            $dbh->do( "REPLACE INTO email_aliases VALUES (?, ?)",
> Not sure I understand your question right, but the email_alias command alias argument doesn't have to be a username (current or redirected), so in that case, you could
> 
> email_alias delete abc_123
> 
> and or
> 
> email_alias delete abc-123
> 
> depending on which alias(es) remained hanging around.
> 
> My reasoning is that whoever uses that command should know what they're doing. Does that answer your question?
> 
> —
> Reply to this email directly or view it on GitHub.
>